lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+  attr_reader :dishes
+  def initialize
+    @dishes = [
+      {number: 1, dish: 'Chicken Biryani', price: 8.99},
+      {number: 2, dish: 'Vegetable Dansak', price: 7.99},
+      {number: 3, dish: 'Rogan Josh', price: 9.49},
+      {number: 4, dish: 'Chicken Korma', price: 9.99}
+    ]
```

Comment:
You are hardcoding the dishes. It might be useful to inject the dishes as a dependency so that in the future you can implement a different menu.